setup.py

```diff
@@ -45,5 +51,13 @@
         "defconQt.representationFactories",
         "defconQt.util",
     ],
-    package_dir={"": "Lib"}
+    scripts=["trufont"],
```

Comment:
It's the `__main__.py` module I guess, though that one does not contain any callable `main()` function you can hook to...
Setuptools wants a function name (with no arguments) in order to make a console script wrapper. 
I don't have a computer at hand now, sorry.
I can try to help tomorrow morning. Cheers!